### PR TITLE
Update the description of the commandline switch `-s`

### PR DESCRIPTION
**Pull request by TheOneRing**
Fixes: #10996

```
Usage: C:\CraftRoot\download\git\owncloud\owncloud-client\cmake-build-debug\bin\owncloud.exe [options] [<vfs file>]
ownCloud version 5.0.0-git
File synchronization desktop utility.


For more information, see https://www.owncloud.com

Options:
  -?, -h, --help         Displays help on commandline options.
  --help-all             Displays help including Qt specific options.
  -v, --version          Displays version information.
  -s, --show             Start with the main window visible, or if it is already running, bring it to the front. By default, the client launches in the background.
  -q, --quit             Quit the running instance.
  --logfile <filename>   Write log to file (use - to write to stdout).
  --logdir <name>        Write each sync log output in a new file in folder.
  --logflush             Flush the log file after every write.
  --logdebug             Output debug-level messages in the log.
  --language <language>  Override UI language.
  --list-languages       Lists available translations, see --language.
  --confdir <dirname>    Use the given configuration folder.
  --debug                Enable debug mode.

Arguments:
  vfs file               Virtual file system file to be opened (optional).
```

### CODE REVIEW

File: src/gui/main.cpp [hunk 1]
```diff
@@ -122,7 +122,12 @@ CommandLineOptions parseOptions(const QStringList &arguments)
         return option;
     };
 
-    auto showSettingsOption = addOption({{QStringLiteral("s"), QStringLiteral("showsettings")}, QStringLiteral("Show the settings dialog while starting.")});
+    auto showSettingsLegacyOption = QCommandLineOption{{QStringLiteral("showsettings")}, QStringLiteral("Hidden legacy option")};
+    showSettingsLegacyOption.setFlags(QCommandLineOption::HiddenFromHelp);
+    parser.addOption(showSettingsLegacyOption);
+
+    auto showOption =
+        addOption({{QStringLiteral("s"), QStringLiteral("show")}, QStringLiteral("Show the main window. By default the client is started in the background.")});
```
Comment (fmoc) [inline src/gui/main.cpp:130]: The second could be misleading to users who do not know the flag. The flag can be used to bring the application to front if running, or to launch it with the settings window visible after startup. We should either not detail this (as it used to be) or make sure it's complete.

**Review by fmoc — CHANGES_REQUESTED** [review]

File: src/gui/main.cpp (hunk 1 above)
Comment (TheOneRing) [reply src/gui/main.cpp:130]: Please propose a better description.

Comment (fmoc) [reply src/gui/main.cpp:130]:
```suggestion
        addOption({{QStringLiteral("s"), QStringLiteral("show")}, QStringLiteral("Start with the main window visible, or if it is already running, bring it to the front. By default, the client launches in the background.")});
```

**Review by fmoc — APPROVED** [review]

**Comment by HanaGemela** [comment]
@TheOneRing anything to test here?

**Comment by HanaGemela** [comment]
Passed in ownCloud 5.0.0.11523-alpha.1

**Pull request merged**